oct/cli/package/common_options.py

```diff
@@ -13,22 +13,44 @@ def package_options(func):
     :param func: Click CLI command to decorate
     :return: decorated CLI command
     """
+    option(
+        '--stage',
+        '-s',
+        'upgrade_stage',
+        type=Choice(['current', 'next', 'fork', 'base']),
+        help='Update the current stage (--stage=current), upgrade to next default stage (--stage=next), or choose the next stage (base, fork)',
+    )(func)
+    option(
+        '--upgrade',
```

Comment:
If kept in a single option it's either true or false so I cannot know whether somebody actually used --upgrade or not.